Userland/DevTools/HackStudio/HackStudioWidget.h

```diff
@@ -153,7 +153,12 @@ class HackStudioWidget : public GUI::Widget {
     RefPtr<Threading::Thread> m_debugger_thread;
     RefPtr<EditorWrapper> m_current_editor_in_execution;
 
-    RefPtr<GUI::Action> m_new_file_action;
+    RefPtr<GUI::Action> m_new_plain_file_action;
+    RefPtr<GUI::Action> m_new_cpp_file_action;
+    RefPtr<GUI::Action> m_new_header_file_action;
+    RefPtr<GUI::Action> m_new_html_file_action;
+    RefPtr<GUI::Action> m_new_javascript_file_action;
```

Comment:
How about an entirely different, crazy idea?
```suggestion
    NonnullRefPtrVector<GUI::Action> m_new_file_actions;
```

(and update all the `foo.add_action(*m_bar_file_action)` to loop over this and add each)

Reply:
I was thinking eventually maybe this wants to be a data file that's read in. Making it a vector would help with that too.